### ISSUE TITLE
Failed to login "did you forget the account?"

### ISSUE THREAD

**Issue by devkarim** [carpedm20/fbchat#581]
## Description of the problem
Unable to login, tried 2 accounts and both raise the same exception.

## Code to reproduce
```py
from fbchat import Session
client = Session.login(email, password)
```

## Traceback
```
Traceback (most recent call last):
  File "session.py", line 36, in get_session
    session = Session.login(
  File "lib\site-packages\fbchat\_session.py", line 293, in login
    raise _exception.NotLoggedIn(error)
fbchat.NotLoggedIn: did you forget the account?
```

## Environment information
- Python version 3.8.2
- `fbchat` version 2.0.0a3

Printed out the headers (might be useful):
```
{'Content-Encoding': 'gzip', 'Cache-Control': 'private, no-cache, no-store, must-revalidate', 'Pragma': 'no-cache', 'Strict-Transport-Security': 'max-age=15552000; preload; includeSubDomains', 'Vary': 'Accept-Encoding', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '0', 'Expires': 'Sat, 01 Jan 2000 00:00:00 GMT', 'X-Frame-Options': 'DENY', 'Content-Type': 'text/html; charset="utf-8"', 'X-FB-Debug': 'AiHQmxiYz5upg/szFdgTF2XdkHqemnR/Ll8F70b+IKNJg7kA+Yiv2a0BnEH0r1KEH10h31WTmdBG6+k0st1mrg==', 'Date': 'Mon, 25 May 2020 20:09:39 GMT', 'Alt-Svc': 'h3-27=":443"; ma=3600', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive'}
```

I used to log into that account several times but not anymore now. I even copied and pasted email and password from the browser just in case they were wrong but no luck so far. I'd really appreciate it if someone could help, thanks.

**Note:** The error message "did you forget the account?" is translated into English as it was in another language.

**Comment by wetmore** [comment]
I got a similar error. Try switching to 2.0.0a2 and see if that fixes it (`pip install fbchat==2.0.0a2`). 

See my comment on https://github.com/carpedm20/fbchat/issues/413 for some context.

**Comment by devkarim** [comment]
Thanks!

> I got a similar error. Try switching to 2.0.0a2 and see if that fixes it (`pip install fbchat==2.0.0a2`).
> 
> See my comment on #413 for some context.

Thank you man, works so well now!
